src/data_collector.py

```diff
@@ -0,0 +1,52 @@
+import aiohttp
+import asyncio
+import boto3
+import json
+
+STATUS_NODES = [
+    'node.explorer.hathor.network',
+    'node1.mainnet.hathor.network',
+    'node2.mainnet.hathor.network'
+]
+
+def process_node_status(node_status: dict):
+    session = boto3.session.Session()
+    lamb = session.client("lambda", endpoint_url="http://localhost:3002")
+    lamb.invoke(
+        InvocationType='Event',
+        FunctionName='hathor-explorer-service-dev-network_data_aggregator',
+        Payload=json.dumps(node_status)
+    )
+
+async def request_node(session: aiohttp.ClientSession, url: str):
+    try:
+        async with session.get(url) as resp:
+            status = await resp.json()
+            process_node_status(status)
```

Comment:
Why would we invoke a lambda when we could just do it here?

Reply:
Lambda will do the aggregation. The idea is to avoid processing as much as possible in the daemon. Just send to lambda and let it go. Network data will be only the first thing to collect. There are three more at least.